cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -183,11 +218,17 @@ label:deprecated[]
 ----
 CREATE [BTREE] INDEX
 ...
-OPTIONS "{" indexConfig: btree-spatial-setting[, ...], indexProvider: btree-provider "}"
+OPTIONS "{" indexProvider: 'native-btree-1.0', indexConfig: btree-spatial-setting[, ...] "}"
 ----
 a|
 B-tree indexes are deprecated and need to be migrated before upgrading to 5.0.
 
+B-tree indexes with the `native-btree-1.0` index provider and spatial index configuration settings are replaced by:
+[source, cypher, role="noheader"]
+----
+CREATE POINT INDEX ...
+----
```

Comment:
should this be replaced by range + point, since point don't cover the rest of the property values?

Reply:
same thing here that they might want one or both